### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by Abecker37**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [x] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [ ] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [x] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [x] My `Javascript` generate no new console errors
- [ ] I tested my code cross browsers
- [x] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [x] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

File: packages/frontend/web/src/components/Navbar.tsx
Comment (bugBuster42) [inline packages/frontend/web/src/components/Navbar.tsx:1]: peux -être mettre les titre du dropdown au singulier, comme tu as fait avec "My information"

**Review by bugBuster42 — APPROVED** [review]

**Review by herve-perel — DISMISSED** [review]

File: packages/frontend/web/src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        >
+          <div className='flex flex-col'>
+            <img src={profil} alt='profil' className='h-7 w-full' />
+            <p className='invisible text-center sm:visible md:visible'>
+              {'Profil'}
+            </p>
+          </div>
+          <img src={arrowDown} alt='arrow-down' className='ml-1 mr-12 mt-4' />
+          {isOpen ? (
+            <div className='aria-hidden absolute right-2 top-14 mt-2 w-60 rounded-md border bg-[#008689]'>
```
Comment (JorisPineau) [inline packages/frontend/web/src/components/Navbar.tsx:63]: #008689 variable color

**Review by JorisPineau — APPROVED** [review]

**Review by JorisPineau — DISMISSED** [review]

**Review by bugBuster42 — APPROVED** [review]

File: packages/frontend/web/src/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import { Link } from 'react-router-dom';
+
+import arrowDown from '../assets/icons/arrow-down.svg';
+import home from '../assets/icons/home.svg';
+import logo from '../assets/icons/logo.svg';
+import message from '../assets/icons/message.svg';
+import profil from '../assets/icons/profil.svg';
+import publish from '../assets/icons/publish.svg';
+import rightArrow from '../assets/icons/right-arrow.svg';
+import trip from '../assets/icons/trip.svg';
```
Comment (YDrogen) [inline packages/frontend/web/src/components/Navbar.tsx:11]: Une raison de pas les mettre dans le dossier `public` ?

File: packages/frontend/web/src/components/Navbar.tsx
Comment (YDrogen) [reply packages/frontend/web/src/components/Navbar.tsx:1]: C'est pas mieux au pluriel justement ?

**Review by YDrogen — CHANGES_REQUESTED** [review]

**Review by JorisPineau — APPROVED** [review]

**Review by herve-perel — APPROVED** [review]

**Review by bugBuster42 — APPROVED** [review]

**Review by herve-perel — DISMISSED** [review]

**Review by JorisPineau — APPROVED** [review]

**Review by bugBuster42 — APPROVED** [review]

**Comment by YDrogen** [comment]
Il suffit pas de resolve les conversations pour les valider

**Review by herve-perel — APPROVED** [review]

File: packages/frontend/web/src/globals.css [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
 
-body {
-  height: 100dvh;
-  width: 100dvw;
-  margin: 0;
-  padding: 0;
-  background: linear-gradient(180deg, #a3b7c8 0%, #5a9891 100%);
+@layer base {
+  :root {
+    --secondary: #0c8587;
```
Comment (JorisPineau) [inline packages/frontend/web/src/globals.css]: primary

File: packages/frontend/web/src/globals.css [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
+    --light: #f9f9f9;
+    --dark: #080808;
+    --grey: #494d55;
```
Comment (JorisPineau) [inline packages/frontend/web/src/globals.css]: secondary

**Review by herve-perel — DISMISSED** [review]

**Review by JorisPineau — DISMISSED** [review]

**Review by herve-perel — APPROVED** [review]

**Review by YDrogen — CHANGES_REQUESTED** [review]
Pourquoi tout le theme se trouve dans cette PR ?

**Review by JorisPineau — APPROVED** [review]

File: packages/frontend/web/tailwind.config.js
Comment (YDrogen) [inline packages/frontend/web/tailwind.config.js:1]: image
Y'a pas un potentiel problème avec `plugins` ?

File: packages/frontend/web/src/globals.css [hunk 5]
```diff
@@ -1,10 +1,33 @@
...
 
-body {
-  height: 100dvh;
-  width: 100dvw;
-  margin: 0;
-  padding: 0;
+@layer base {
+  :root {
+    --primary: #0c8587;
+    font-family: 'Inter', sans-serif;
```
Comment (YDrogen) [inline packages/frontend/web/src/globals.css:9]: https://tailwindcss.com/docs/font-family

**Review by YDrogen — CHANGES_REQUESTED** [review]

**Review by bugBuster42 — APPROVED** [review]

**Review by herve-perel — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**